File: backend/src/main/java/com/crm/CRMBackend/models/Response.java
package com.crm.CRMBackend.models;

import java.time.LocalDateTime;

import org.springframework.stereotype.Component;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Component
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Response {
	
	private int id;
	private int agentId;
	private int customerId;
	private int ticketId;
	private String message;
	private String ticketStatus;
	private LocalDateTime createdTime;
	private boolean responseByAgent;
	
}

File: frontend/src/app/models/Response.ts


export interface Response {
    id: number,
    agentId: number,
    agentName: string,
    customerId: number,
    message: string,
    ticketStatus: string,
    createdTimestamp: string,
    responseByAgent: boolean,
    customerName: string,
    companyName: string
}